server/chat-commands/core.ts

```diff
@@ -394,7 +394,7 @@ export const crqHandlers: {[k: string]: Chat.CRQHandler} = {
 			autoconfirmed: !!targetUser.autoconfirmed,
 			status: targetUser.getStatus(),
 			rooms: roomList,
-			friended: user.friends?.has(targetUser.id),
+			friended: user.id === targetUser.id ? undefined : user.friends?.has(targetUser.id),
```

Comment:
```ts
friended: user.friends?.has(targetUser.id) || undefined,
```

would do the same thing for all people who aren't friends, for more bandwidth savings.

Reply:
Same goes for `status` and `autoconfirmed`.

Reply:
I did this.